addons/common/config.cpp

```diff
@@ -42,3 +47,183 @@ class RscDisplayCurator {
         };
     };
 };
+class CfgVideoOptions

```

Comment:
Can you move this into CfgVideoOptions and #include it instead?

Reply:
Done